addons/web/static/tests/views/list_view_tests.js

```diff
@@ -16398,4 +16412,786 @@ QUnit.module("Views", (hooks) => {
+            if (record.m2o === 1) {
+                record.properties = [{ ...definition0, value: "AA" }];
+            }
+        }
+
+        await makeView({
+            type: "list",
+            resModel: "foo",
+            serverData,
+            arch: `
+                <tree editable="bottom">
+                    <field name="properties" />
+                </tree>
+            `,
+            mockRPC(route) {
+                assert.step(route);
+            },
+            irFilters: [
+                {
+                    context: "{}",
+                    domain: "[['id', '=', 1]]",
+                    id: 7,
+                    name: "only one",
+                    sort: "[]",
+                    user_id: [2, "Mitchell Admin"],
+                },
+            ],
+        });
+
+        assert.verifySteps([
+            "/web/dataset/call_kw/foo/get_views",
+            "/web/dataset/call_kw/foo/search_read",
+            "/web/dataset/call_kw/bar/read",
+            "/web/dataset/call_kw/foo/web_search_read",
+        ]);
+
+        await toggleFavoriteMenu(target);
+        await toggleMenuItem(target, "only one");
+
+        assert.verifySteps([
+            "/web/dataset/call_kw/foo/search_read",
+            "/web/dataset/call_kw/bar/read",
+            "/web/dataset/call_kw/foo/web_search_read",
+        ]);
+    });
+
+    QUnit.test("do not reload properties definitions when page change", async (assert) => {
+        const definition0 = {
+            type: "char",
+            name: "property_char",
+            string: "Property char",
+        };
+        serverData.models.bar.records[0].definitions = [definition0];
+        for (const record of serverData.models.foo.records) {
+            if (record.m2o === 1) {
+                record.properties = [{ ...definition0, value: "0" }];
+            }
+        }
+
+        await makeView({
+            type: "list",
+            resModel: "foo",
+            serverData,
+            arch: `
+                <tree editable="bottom">
+                    <field name="properties" />
+                </tree>
+            `,
+            mockRPC(route) {
+                assert.step(route);
+            },
+            irFilters: [
+                {
+                    context: "{}",
+                    domain: "[['id', '=', 1]]",
+                    id: 7,
+                    name: "only one",
+                    sort: "[]",
+                    user_id: [2, "Mitchell Admin"],
+                },
+            ],
+        });
+
+        assert.verifySteps([
+            "/web/dataset/call_kw/foo/get_views",
+            "/web/dataset/call_kw/foo/search_read",
+            "/web/dataset/call_kw/bar/read",
+            "/web/dataset/call_kw/foo/web_search_read",
+        ]);
+
+        await toggleFavoriteMenu(target);
+        await toggleMenuItem(target, "only one");
+
+        assert.verifySteps(["/web/dataset/call_kw/foo/web_search_read"]);
```

Comment:
wrong, this is duplicated from the test "reload properties definitions when domain change"